src/OptiHPLCHandler/empower_api_core.py

```diff
@@ -115,47 +133,85 @@ def logout(self) -> None:
         self.session_id = None
         logger.debug("Logout successful")
 
-    def get(self, endpoint: str) -> requests.Response:
+    def _requests_wrapper(
+        self, method: str, endpoint: str, body: Optional[dict], timeout
+    ) -> requests.Response:
         """
-        Get data from Empower.
+        Wrapper for requests.
 
-        :param endpoint: The endpoint to get data from.
+        :param method: The method to use.
+        :param endpoint: The endpoint to use.
+        :param body: The body to use.
         """
+
+        def _request_with_timeout(method, endpoint, header, body, timeout):
+            try:
+                response = requests.request(
+                    method,
+                    endpoint,
+                    json=body,
+                    headers=header,
+                    timeout=timeout,
+                )
+            except requests.exceptions.Timeout as e:
+                print(f"{method}ing {body} to {endpoint} timed out")
```

Comment:
Again with defining the same string multiple times.

Reply:
fixed